DolbyIO/Source/DolbyIOModule/Private/DolbyIO/SdkAccess.h

```diff
@@ -0,0 +1,56 @@
+// Copyright 2022 Dolby Laboratories
+
+#pragma once
+
+#include "DolbyIO/Typedefs.h"
+
+namespace dolbyio::comms
+{
+	enum class conference_status;
+	class refresh_token;
+	class sdk;
+}
+
+namespace DolbyIO
+{
+	class ISdkEventObserver;
+
+	class FSdkAccess final
+	{
+		using FToken = FString;
+
+	public:
+		using EConferenceStatus = dolbyio::comms::conference_status;
+
+		FSdkAccess(ISdkEventObserver&);
+		~FSdkAccess();
+
+		void SetToken(const FToken&);
+
+		void Connect(const FString& ConferenceName, const FString& UserName);
+		void Disconnect();
+
+		void UpdateViewPoint(const FVector&, const FRotator&);
+
+		void MuteInput();
+		void UnmuteInput();
+		void MuteOutput();
+		void UnmuteOutput();
+
+		void GetAudioLevels();
+
+	private:
+		void Initialize(const FToken&);
+		bool IsConnected() const;
+		void UpdateStatus(EConferenceStatus);
+		class FErrorHandler MakeHandler(int Id);
+
+		ISdkEventObserver& Observer;
+		TUniquePtr<dolbyio::comms::sdk> Sdk;
+		TUniquePtr<dolbyio::comms::refresh_token> RefreshTokenCb;
+		EConferenceStatus ConferenceStatus;
+		FParticipant LocalParticipantID;
```

Comment:
FParticipant looks like an oversimplification of participant representation, the UUID format alone can only uniquely identify a remote participant but the application will not know who that participant is. In order to be more complete we will need things like external name, ID and avatar which are used when we open session. So I would say at a minimum:
- expose external ID in the `connect` function
- include external ID in the participant representation
- external ID is a way for developers to uniquely identify an user *before* Comms API maps to an internal represenation of the user ID in UUID format. 
- then FParticipant should be extended to include external ID and user name. This way it'd be very easy for the app to figure out who a remote participant is.

Reply:
Note token. I will apply relevant changes.

Reply:
Please verify whether the information I introduced satisfy these requirements.